### PR TITLE
IMOD-946: add

### PR DESCRIPTION
**Pull request by HendrikKok**
function to add correction-term to the mf6 wel-package

### CODE REVIEW

File: imod_coupler/drivers/dfm_metamod/mf6_wrapper.py [hunk 1]
```diff
@@ -38,3 +38,36 @@ def set_river_stages(
...
+        Raises
+        ------
+        ValueError
+            the size of the provided flux array does not match the expected size
+        """
+        bound_adress = self.get_var_address("BOUND", mf6_flowmodel_key, mf6_wel_pkg_key)
+        bound = self.get_value_ptr(bound_adress)
+        flux = bound[:, 0]
+        if correction_flux is None or len(correction_flux) != len(flux):
+            raise ValueError(f"Expected size of new_river_stages is {len(flux)}")
```
Comment (luitjansl) [inline imod_coupler/drivers/dfm_metamod/mf6_wrapper.py:71]: new_river_stages =>correction_flux

**Review by luitjansl — APPROVED** [review]
looks good, but you should also add a (unit) test for this.

**Comment by Hofer-Julian** [comment]
Also the tests seem to be red.

File: imod_coupler/drivers/dfm_metamod/mf6_wrapper.py (hunk 1 above)
Comment (HendrikKok) [reply imod_coupler/drivers/dfm_metamod/mf6_wrapper.py:71]: done

**Pull request merged**